tests/Integration/inc/functions/getRocketCacheRejectUri.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\functions;
+
+use Brain\Monkey\Functions;
+use WPMedia\PHPUnit\Integration\TestCase;
+
+/**
+ * @covers ::get_rocket_cache_reject_uri
+ * @group Functions
+ * @group Options
+ */
+class Test_GetRocketCacheRejectUri extends TestCase {
+
+	/**
+	 * @dataProvider providerTestData
+	 */
+	public function testShouldReturnExcludeDeferJSArray( $config, $expected ) {
+		$options = [
+			'cache_reject_uri' => $config['options']['cache_reject_uri'],
+		];
+		update_option( 'wp_rocket_settings', $options );
```

Comment:
For the options part, we have the `SettingsTrait` available to make sure everything is saved/restored to original on setup/teardown